packages/firestore/src/platform_node/node_platform.ts

```diff
@@ -75,4 +77,10 @@ export class NodePlatform implements Platform {
   btoa(raw: string): string {
     return new Buffer(raw, 'binary').toString('base64');
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    validatePositiveNumber('randomBytes', 1, nBytes);
```

Comment:
DebugAssert here as well.

Reply:
Also, make sure that `0` is accepted as a valid value for `nBytes`.